deployment/dev_env/docker-compose.yml

```diff
@@ -9,6 +10,11 @@ services:
       - "8400"
       - "8500:8500"
       - "53"
+    logging:
```

Comment:
Can we reuse this config? You don't need x-logging, you can do it as shown here: https://stackoverflow.com/questions/38567355/docker-compose-global-level-logging